### PR TITLE
Making RHEL7 conductor

### PR DESCRIPTION
**Pull request by chouseknecht**
##### ISSUE TYPE
 - Feature Pull Request

##### SUMMARY
Contains changes required to get a conductor from RHEL7:latests built, passing tests and scanned.

### CODE REVIEW

File: container/docker/templates/conductor-src-dockerfile.j2 [hunk 1]
```diff
@@ -24,7 +24,8 @@ RUN yum -y update-minimal --disablerepo "*" \
                    https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
     yum -y install --disablerepo "*" \
                    --enablerepo epel,rhel-7-server-rpms,rhel-7-server-optional-rpms \
-                   make gcc git python-devel curl rsync libffi-devel openssl-devel tar redhat-rpm-config&& \
+                   make gcc git python-devel curl rsync libffi-devel openssl-devel tar redhat-rpm-config && \
+    yum -y remove python-chardet && \
```
Comment (j00bar) [inline container/docker/templates/conductor-src-dockerfile.j2]: What's with this removal?

Comment (chouseknecht) [reply container/docker/templates/conductor-src-dockerfile.j2]: No idea why, but the RPM check within the scanner flags the existence of the`chardet` module files within `/usr/lib/python2.7/site-pacakges/`, and marks the RPM test as failed. Removing it mollifies the scanner, and doesn't seem to break our tests.

Comment (j00bar) [reply container/docker/templates/conductor-src-dockerfile.j2]: I'd love to understand that.

**Comment by j00bar** [comment]
It's awesome we're writing tests - any way we can do these tests as part of the normal `run_tests.yml`?

**Comment by chouseknecht** [comment]
*awesome* is a subjective word, but point taken.

**Pull request merged**